File: Operation/DecodeBase58/DecodeBase58.py

import base64
import binascii

# 要解码的Base64字符串
base64_str = "f/8fwAM+ww8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="

# 解码为字节数组
byte_data = base64.b64decode(base64_str)

# 转换为Hex字符串
hex_str = binascii.hexlify(byte_data).decode('utf-8')

print(hex_str)

File: Operation/DecodeBase58/DecodeBase58.js
// 要解码的Base64字符串
const base64Str = "f/8fwAM+ww8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";

// 将Base64解码为字节数组
const buffer = Buffer.from(base64Str, 'base64');

// 将字节数组转换为Hex字符串
const hexStr = buffer.toString('hex');

console.log(hexStr);